src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Cells/HealthCertificateQRCodeCellViewModel.swift

```diff
@@ -24,6 +24,7 @@ struct HealthCertificateQRCodeCellViewModel {
 			showRealQRCodeIfValidityStateBlocked: mode == .details,
 			imageAccessibilityTraits: .image,
 			accessibilityLabel: accessibilityText,
+			qrCodeAccessibilityID: AccessibilityIdentifiers.HealthCertificate.qrCodeView(of: healthCertificate.uniqueCertificateIdentifier),
```

Comment:
Please rename it to accessibilityIdentifier everywhere in order to be consistent. 

```suggestion
			qrCodeAccessibilityIdentifier: AccessibilityIdentifiers.HealthCertificate.qrCodeView(of: healthCertificate.uniqueCertificateIdentifier),
```